packages/client/src/svelte/components/Main/Tabs/Orders/OrderItem.svelte

```diff
@@ -34,8 +34,8 @@
 
   const staticMaterial = $staticContent.materials.find(
     material =>
-      material.materialType === MATERIAL_TYPE[order.order.materialType],
-  )
+      material.materialType === name,
```

Comment:
This seems separate from the main project, leaving as-is for compatibility
You could then refactor names to be actually human-readable (in the materials `init`), instead of keeping the underscored enum labels